ql/src/java/org/apache/hadoop/hive/ql/QueryPlan.java

```diff
@@ -20,6 +20,7 @@
 
 import java.io.IOException;
 import java.io.Serializable;
+import java.nio.charset.StandardCharsets;
```

Comment:
no changes here, could we revert?

Reply:
realized this was refactored by the IDE, now it's correct alphabetical ordering point of view, can I leave it like this?